src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ApiServiceActor.scala

```diff
@@ -55,11 +57,32 @@ class ApiServiceActor extends HttpServiceActor with LazyLogging {
   val methodsService = new MethodsService() with ActorRefFactoryContext
   val configurationsService = new ConfigurationsService() with ActorRefFactoryContext
 
+  def withResourceFileContents(path: String)(innerRoute: String => Route): Route =
+    innerRoute( FileUtils.readAllTextFromResource(path) )
 
   def possibleRoutes =  options{ complete(OK) } ~ methodsService.routes ~ configurationsService.routes ~
     get {
       pathSingleSlash {
-        getFromResource("swagger/index.html")
+        withResourceFileContents("swagger/index.html") { indexHtml =>
+          complete {
+            val swaggerOptions =
+              """
+                |        validatorUrl: null,
+                |        apisSorter: "alpha",
+                |        operationsSorter: "alpha",
+              """.stripMargin
+            HttpEntity(ContentType(MediaTypes.`text/html`),
+              indexHtml
+                .replace("your-client-id", AgoraConfig.SwaggerConfig.clientId)
+                .replace("your-realms", AgoraConfig.SwaggerConfig.realm)
+                .replace("your-app-name", AgoraConfig.SwaggerConfig.appName)
+                .replace("scopeSeparator: \",\"", "scopeSeparator: \" \"")
+                .replace("jsonEditor: false,", "jsonEditor: false," + swaggerOptions)
+                .replace("url = \"http://petstore.swagger.io/v2/swagger.json\";",
+                  "url = '/api-docs';")
```

Comment:
The api-docs works for me locally. There is an endpoint in agora that does this so I think both cases will work.